csharp/src/Google.Protobuf/Reflection/FieldDescriptor.cs

```diff
@@ -197,7 +197,7 @@ public OneofDescriptor ContainingOneof
         }
 
         /// <summary>
-        /// Returns  the type of the field.
+        /// Returns the type of the field.
```

Comment:
Note that this file change was just a typo fix that was simpler to include here than put into its own PR...